napalm_logs/listener/base.py

```diff
@@ -8,7 +8,7 @@ class ListenerBase:
     '''
     The base class for the listener.
     '''
-    def __init__(self, addr, port):
+    def __init__(self, **kwargs):
```

Comment:
If we do this, inside the listener class we must verify that the expected fields are configured.